cmd/noisd/wasmvm.go

```diff
@@ -0,0 +1,9 @@
+//go:build wasmvm
+// +build wasmvm
+
+package main
+
+// This is a dummy file to make sure the wasmvm is imported as direct dependency
+import (
+	_ "github.com/CosmWasm/wasmvm"
+)
```

Comment:
Without that file, is something complaining about the direct dependency?

In https://github.com/CosmosContracts/juno/pull/642 they bumped the indirect dependency but I wonder if this is safe.

Reply:
I think is not necessary let me try.

Reply:
Do we need those source code additions?

Reply:
tested it without this seems like it's respecting the new version